.github/workflows/performance-test.yml

```diff
@@ -0,0 +1,85 @@
+name: performance-test
+on:
+  push:
+    branches: [ automate_performance_testing ]
```

Comment:
We'd like to trigger the test on "new tags on release branches `release-*`", can you please update accordingly?